style.css

```diff
@@ -468,6 +511,7 @@ ul.skills-list li:first-child {
 .skills-group {
   display: flex;
   flex-direction: column;
+  height: 100%;
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible. Fix the spacing from left and right to match the Figma design. Use less space in between and more from left and right.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213360517-cfae5888-33ab-46c6-a4e2-38982a8ca1c5.jpg)


### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/213360535-51d6c200-9133-45b9-9c90-4eed5fab2f12.jpg)